dist/s_clang-scan.diff

```diff
@@ -1,23 +1,120 @@
+src/block/block_ckpt_scan.c:273:14: warning: Although the value stored to 'ret' is used in the enclosing expression, the value is never actually read from 'ret'
```

Comment:
I guess the contents of this file were generated when running a clang scan against the current codebase. I'm curious about the criteria we use to determine accepting/rejecting warnings/errors.

Reply:
Never mind. I just saw you created a follow-on ticket WT-8052.

Reply:
The quick solution at this time is to accept the generated clang warnings, however, a proper solution would be to improve the code, (or at least make a reasonable conclusion if the final decision is to accept the warnings). 
I have created a ticket for this WT-8052.